ci/e2e-tests.sh

```diff
@@ -41,12 +41,12 @@ get_package() {
     # Get workflow runs for this branch and pick the latest one (the first one).
     # From that, get the artifacts URL.
     #
-    # Ref: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#list-workflow-runs
+    # Ref: https://docs.github.com/en/rest/reference/actions#list-workflow-runs
     echo "Getting latest workflow run's artifacts URL..." >&2
     artifacts_url="$(
         github_curl -L \
             -H 'accept: application/vnd.github.v3+json' \
-            "$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/actions/workflows/packages.yaml/runs?branch=$BRANCH&event=push&status=success" |
+            "$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/actions/workflows/packages.yaml/runs?branch=${BRANCH//\//%2f}&event=push&status=success" |
```

Comment:
Out of curiosity, what's this change all about?

Reply:
bash-ified `encodeURIComponent`